lib/core/migrate/migrate.js

```diff
@@ -513,6 +513,12 @@ const migrations = [
 		},
 	},
 
+	{ // FIXME Place correctly in migrator list
```

Comment:
I'm not sure what you mean here; this PR looks good so it can be probably be merged for the first 5.7.0 beta, so the migrator can probably go right here